docs/sql.md

```diff
@@ -16,6 +16,17 @@ sql:
 ---
 ```
 
+To load externally-hosted data, you can use a full URL:
+
+```yaml
+---
+sql:
+  quakes: https://earthquake.usgs.gov/earthquakes/feed/v1.0/summary/all_day.csv
+---
+```
+
+<div class="tip">For performance and reliability, we recommend using local files rather than loading data from external servers at runtime. If needded, you can use a <a href="./loaders">data loader</a> to take a snapshot of a remote data during build.</div>
```

Comment:
```suggestion
<div class="tip">For performance and reliability, we recommend using local files rather than loading data from external servers at runtime. If needed, you can use a <a href="./loaders">data loader</a> to take a snapshot of a remote data during build.</div>
```